projects/govuk-account-manager-prototype/docker-compose.yml

```diff
@@ -0,0 +1,66 @@
+version: '3.7'
+
+x-govuk-account-manager-prototype: &govuk-account-manager-prototype
+  build:
+    context: .
+    dockerfile: Dockerfile.govuk-base
+  image: govuk-account-manager-prototype
+  stdin_open: true
+  tty: true
+  volumes:
+    - ${GOVUK_ROOT_DIR:-~/govuk}:/govuk:delegated
+    - home:/home/build
+  working_dir: /govuk/govuk-account-manager-prototype
+
+services:
+  govuk-account-manager-prototype-lite:
+    <<: *govuk-account-manager-prototype
+    privileged: true
+    depends_on:
+      - postgres
+      - memcached
+      - govuk-attribute-service-prototype-app
+    environment:
+      MEMCACHE_SERVERS: memcached
+      DATABASE_URL: "postgresql://postgres@postgres/govuk-account-manager-prototype"
+      TEST_DATABASE_URL: "postgresql://postgres@postgres/govuk-account-manager-prototype-test"
+      REDIRECT_BASE_URL: http://www.login.service.dev.gov.uk
+      FINDER_FRONTEND_OAUTH_CLIENT_ID: transition-checker-id
+      FINDER_FRONTEND_OAUTH_CLIENT_PUBLIC_KEY_UUID: 898d62b7-eed9-464a-a4ae-9d9e08bd9bee
+      FINDER_FRONTEND_OAUTH_CLIENT_PUBLIC_KEY: "-----BEGIN PUBLIC KEY-----\nMFkwEwYHKoZIzj0CAQYIKoZIzj0DAQcDQgAEjMgE/d6Bdu1K17BTjNycDvIXKSmU\ngZ5YYoayU3gGqTLhgefuK2qbo99Wx+SuvdW/GRvIlUIW1ooNRdQ3QFwwCw==\n-----END PUBLIC KEY-----\n"
```

Comment:
I slightly feel we should add the corresponding finder-frontend env vars in this same commit, so it's clear which private key this public key belongs to.  #417 can just add the `make` dependency, the `docker-compose` dependency, and the `FEATURE_FLAG_ACCOUNTS: "enabled"` env var

But it's up to you.

Reply:
So my thinking here on leaving `FEATURE_FLAG_ACCOUNTS: "enabled"` out was, that I didn't want to force anyone on govuk-docker `main` to have to have access to our private repos or find their developent version showing our stuff before it's live?

I've been thinking about [renaming these](https://github.com/alphagov/govuk-docker/pull/417/files#diff-81716fac3ebcda04e6914c20d3333c82R38-R40) so they correspondbetter to the ones above?
Do you think that works?

Reply:
Yes, I think we should keep `FEATURE_FLAG_ACCOUNTS: "enabled"` of this PR; but I'm not sure about the other environment variables, as this PR introduces a public key with no corresponding private key.

Reply:
Renaming them would be good, as the `FINDER_FRONTEND_OAUTH_CLIENT_PUBLIC_KEY_UUID` and `FINDER_FRONTEND_OAUTH_CLIENT_PUBLIC_KEY` don't really have anything to do with OAuth

Reply:
ah right, gotcha - cool will do that now

Reply:
Ok have move these back, and renamed the ones Bruce made.
I'll save the renaming of the private key for the moment as we'll need to make the corresponding changes to `finder-frontend` and andy is trying to get set up of these branches.

So i'll put that in a follow up PR?

Reply:
I know I'm late to the party here, but it's quite alarming to see such long and specific variables being encoded here. I'm assuming these need to be carefully synced to the OAuth provider? Have we considered a more lightweight approach, like [the one used by gds-sso](https://github.com/alphagov/gds-sso#use-in-development-mode)? Sometimes a `.env` file or similar is just more flexible than hard-coding bespoke strings for (in future) all apps.